src/actions/signout.tsx

```diff
@@ -1,5 +1,6 @@
 import { auth } from '../firebase';
 
 export const signout: () => void = () => {
+  // eslint-disable-next-line
```

Comment:
you don't need // eslint-disable-next-line
just return this function then change types

Reply:
this was for the console.log, not the return